pycon/finaid/views.py

```diff
@@ -425,3 +425,28 @@ def get_value(name, object):
         writer.writerow(data)
 
     return response
+
+
+@login_required
+def receipt_upload(request):
+    receipts = request.user.financial_aid_receipts.all()
```

Comment:
I guess if we remove the 'user' field from the receipt model, this'll have to change too.